dashboard/src/components/PackageHeader/PackageHeader.test.tsx

```diff
@@ -42,7 +42,7 @@ const testProps: IPackageHeaderProps = {
 
 it("renders a header for the package", () => {
   const wrapper = mount(<PackageHeader {...testProps} />);
-  expect(wrapper.text()).toContain("testrepo/test");
+  expect(wrapper.text()).toContain("testrepo/foo");
```

Comment:
What's triggering an expectation change here?

Reply:
It is a problem with the availablePackageDetail being passed. In fact, I'm fixing it instead of changing the assertion.

Let  `availablePackageDetail.name` be `test` and `availablePackageDetail.availablePackageRef.identifie` be `testrepo/foo`,
before, we were concatening repo+name, that is `testrepo/test` and, now, we're just using the id, that is `testrepo/foo`

We just should change the availablePackageDetail.name from "test" to "foo".